cirq/protocols/channel.py

```diff
@@ -175,3 +178,19 @@ def has_channel(val: Any) -> bool:
 
     # No has methods, use `_channel_` or delegates instead.
     return channel(val, None) is not None
+
+
+def validate_channel(supports_channel: SupportsChannel, rtol: int = 1e-5, atol: int = 1e-8):
```

Comment:
The type for `supports_channel` isn't quite correct because this also works for channel mixtures.  Suggest changing this to a union type that includes mixture and unitary cases.  Then I'd just call this parameter `channel`.